### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by Dawn-W**
This is Module 2 homework.

### CODE REVIEW

File: CalcCurrentValue/CalcCurrentValue/CalcCurrentValue.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalcCurrentValue
+{
+    class CalcCurrentValue
+    {
+        public enum TransactionType
```
Comment (d-okhrimenko) [inline CalcCurrentValue/CalcCurrentValue/CalcCurrentValue.cs]: It is better to have new types in a separate file. Please move enum from Program.cs to TransactionType.cs. You can add a new file by right clicking on the project name and then selecting "add" > "class..."

File: CalcCurrentValue/CalcCurrentValue/CalcPL.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalcCurrentValue
+{
+    class CalcPL
```
Comment (d-okhrimenko) [inline CalcCurrentValue/CalcCurrentValue/CalcPL.cs]: I think you can rename this class and give more descriptive name

File: CalcCurrentValue/CalcCurrentValue/CalcPL.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            TradePrice = tradeprice;
+            OriginalPrice = originalprice;
+            Nominal = nominal;
+        }
+        public double TradePrice { get; }
+        public double OriginalPrice { get; }
+        public int Nominal { get; }
+
+        public double PL() => (TradePrice - OriginalPrice) * Nominal;
```
Comment (d-okhrimenko) [inline CalcCurrentValue/CalcCurrentValue/CalcPL.cs]: Consider using verbs for method naming. e.g. Calculate, GetResult

File: CalcCurrentValue/CalcCurrentValue/CalcCurrentValue.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalcCurrentValue
+{
+    class CalcCurrentValue
```
Comment (d-okhrimenko) [inline CalcCurrentValue/CalcCurrentValue/CalcCurrentValue.cs]: Try to give another name for this class. It look like name of method not a class.  CurrentValueCalculator more sutable name for a class

**Comment by Dawn-W** [comment]
Fixed naming problems.

**Pull request merged**